torch_np/_ufuncs.py

```diff
@@ -6,13 +6,10 @@
 
 from . import _binary_ufuncs_impl, _helpers, _unary_ufuncs_impl
 from ._detail import _dtypes_impl, _util
-from ._normalizations import ArrayLike, DTypeLike, OutArray, SubokLike, normalizer
+from ._normalizations import ArrayLike, DTypeLike, OutArray, normalizer
 
 
 def _ufunc_preprocess(tensors, where, casting, order, dtype, subok, signature, extobj):
```

Comment:
nb. It'd be better to use the `normalizer` decorator directly when defining the `@wrapped` function, to be more explicit that they're being normalized. It's happened to me a couple times that I forgot that these functions are normalized just find that they are being normalized after calling `decor_{unary,binary}`.

Reply:
```
def decorator(...):
    @another_decorator
    def wrapped(...):
        ....
```

"I put a decorator inside your decorator", wow :-). Readability counts, so done in the updated PR.